src/main/java/edu/kit/ipd/crowdcontrol/objectservice/notification/NotificationPolicy.java

```diff
@@ -0,0 +1,34 @@
+package edu.kit.ipd.crowdcontrol.objectservice.notification;
+
+/**
+ * NotificationPolicy encapsulates strategies to check and send notifications.
+ *
+ * @author Simon Korz
+ * @version 1.0
+ */
+public abstract class NotificationPolicy<T> {
```

Comment:
this is an OOP question...is you class an abstraction or an contract? If you have a contract its an interface (like i am list, i have a get(index), remove(index) etc. method) or an abstraction, like a Thread. its your decision, it never meant forcing you to implement an interface, this was more of a question